src/shared/components/checkbox/Checkbox.scss

```diff
@@ -5,31 +5,27 @@
   grid-template-columns: 27px 1fr;
 }
 
-.defaultCheckbox {
+.checkboxDefault {
   cursor: pointer;
   justify-self: center;
-  height: 14px;
-  width: 14px;
-  border: 1px solid $grey;
-  background-color: $white;
   transition: background-color 0.1s ease-in-out;
   margin-top: 7px;
+  background-color: var(--checkbox-background-color, #{$white});
+  border: 1px solid var(--checkbox-border-color, #{$grey});
+  height: var(--checkbox-dimension-size, 13px);
+  width: var(--checkbox-dimension-size, 13px);
 
   & + label {
     cursor: pointer;
     font-weight: $light;
   }
 }
 
-.unchecked {
-  background-color: $white;
-}
-
-.checked {
+.checkboxChecked {
   background-color: $green;
 }
 
-.disabled {
+.checkboxDisabled {
   background-color: $light-grey;
   cursor: default;
```

Comment:
we may want to add `pointer-events: none`

Reply:
✅